pkg/controller/id.go

```diff
@@ -57,5 +61,5 @@ func (i ID) Uint64() uint64 {
 
 // String returns the identifier as a string
 func (i ID) String() string {
-	return i.Value.(string)
+	return fmt.Sprintf("%s", i.Value)
```

Comment:
@ray-milkey should this be `%d` then? Besides - I think it's only in the tests that the identifier is an Int. In the running `onos-config` strings are used.
e.g. https://github.com/onosproject/onos-config/blob/c18d4ddd23ea0fafb602df37d559412e38e0dfc8/pkg/controller/change/device/controller_test.go#L116